src/basic_recipes/contours.jl

```diff
@@ -45,32 +51,50 @@ $(ATTRIBUTES)
     default_theme(scene, Contour)
 end
 
-function contourlines(::Type{<: Contour}, contours, cols)
-    result = Point2f[]
+angle(p1::Union{Vec2f,Point2f}, p2::Union{Vec2f,Point2f})::Float32 =
+    atan(p2[2] - p1[2], p2[1] - p1[1])  # result in [-π, π]
+
+function label_info(lev, vertices, col)
+    mid = ceil(Int, 0.5f0 * length(vertices))
+    pts = (vertices[max(firstindex(vertices), mid - 1)], vertices[mid], vertices[min(mid + 1, lastindex(vertices))])
+    lev_short = round(lev; digits = 2)
+    (
+        string(isinteger(lev_short) ? round(Int, lev_short) : lev_short),
```

Comment:
It would be nice to allow the user to pass their own formatter through to here, in the same style as `ax.ticklabelformat`.

Reply:
Good idea I'll add that.

Reply:
Implemented.